lunatrace/bsl/backend/src/config.ts

```diff
@@ -124,22 +124,33 @@ export function getQueueHandlerConfig(): QueueHandlerWorkerConfig {
   const DEFAULT_QUEUE_VISIBILITY = 60;
 
   const handlerConfigLookup: Record<QueueHandlerType, QueueHandlerConfig> = {
-    'process-webhook': {
-      maxMessages: 1,
-      visibility: DEFAULT_QUEUE_VISIBILITY,
-      envVar: 'PROCESS_WEBHOOK_QUEUE',
-    },
-    'process-manifest': {
-      maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
-      visibility: DEFAULT_QUEUE_VISIBILITY,
-      envVar: 'PROCESS_MANIFEST_QUEUE',
-    },
-    'process-sbom': {
+    // 'process-webhook': {
+    //   maxMessages: 1,
+    //   visibility: DEFAULT_QUEUE_VISIBILITY,
+    //   envVar: 'PROCESS_WEBHOOK_QUEUE',
+    // },
+    // 'process-manifest': {
+    //   maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+    //   visibility: DEFAULT_QUEUE_VISIBILITY,
+    //   envVar: 'PROCESS_MANIFEST_QUEUE',
+    // },
+    // 'process-sbom': {
+    //   maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+    //   visibility: DEFAULT_QUEUE_VISIBILITY,
+    //   envVar: 'PROCESS_SBOM_QUEUE',
+    // },
+    // 'process-repository': {
+    //   maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
+    //   visibility: DEFAULT_QUEUE_VISIBILITY * 10,
+    //   envVar: 'PROCESS_REPOSITORY_QUEUE',
+    // },
+    // TODO (cthompson) fill these in
+    's3-queue-handler': {
       maxMessages: DEFAULT_QUEUE_MAX_MESSAGES,
-      visibility: DEFAULT_QUEUE_VISIBILITY,
-      envVar: 'PROCESS_SBOM_QUEUE',
+      visibility: DEFAULT_QUEUE_VISIBILITY * 10,
+      envVar: 'PROCESS_REPOSITORY_QUEUE',
     },
-    'process-repository': {
+    'lunatrace-queue-handler': {
```

Comment:
And i guess the implication here is that lunatrace-queue-handler takes events that we have fired, instead of fired from s3?  I would probably call it custom-event-queue-handler or something. Just a little confused because the whole thing is  lunatrace isn't it?  

Not saying we have to change it